src/utils/validators_enhanced.py

```
@@ -0,0 +1,453 @@
+    """Backward compatible email validation function."""
+    try:
+        return bool(validate_email_enhanced(email))
+    except Exception:
+        return False
+
+
+def validate_nickname(nickname: str) -> bool:
+    """Backward compatible nickname validation function."""
+    try:
+        return bool(validate_nickname_enhanced(nickname))
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)